packages/kbn-logging/src/ecs/agent.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * https://www.elastic.co/guide/en/ecs/1.12/ecs-agent.html
+ * https://www.elastic.co/guide/en/ecs/master/ecs-agent.html
```

Comment:
are you going to upgrade it to `8.0` in the next PR? `master` might be out of sync with the definitions in the Kibana repo, so we should prefer a pinned version in the link

Reply:
Yeah we will need to update this in a follow-up as the 8.0 docs aren't available until FF, so all we have is `master`.

As an alternative we could just link now to the 8.0 URLs, which will return a 404 until they are available.